bin/check-cpplint.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+: "${KALEIDOSCOPE_DIR:=$(pwd)}"
+cd "${KALEIDOSCOPE_DIR}"
+
+OPTIONS+=(
```

Comment:
This doesn't match the old logic.  --noisy complained about more stuff than "normal" - it was an aspirational goal we aren't yet hitting.

But also, cpplint supports 'CPPLINT.cfg' -instead of a wrapper script, let's put the standard config in the config file?

Reply:
This is the one I had looked into the least; I hadn't even checked if it could be done with a config file.  That definitely sounds better than what I did!

Reply:
Ugh.  Apparently, `cpplint.py`'s config file(s) take precedence over command-line options, and the config file name is hard-coded, so if want a "noisy" mode and a "quiet" mode, you can only specify the filters that they have in common in the config file.  Or we could use a wrapper that symlinks to the appropriate config file.  Or we could edit `cpplint.py`, which leads me to ask where this version of it came from, because I can't find a version of it that looks like ours in the history here: https://github.com/google/styleguide/tree/gh-pages/cpplint)

Reply:
Ah, I think I found it here: https://github.com/cpplint/cpplint